app/assets/stylesheets/application.scss

```diff
@@ -607,30 +609,40 @@ label.remove_time_span {
 }
 
 .documentHeader {
-  position: relative;
-  h3 {
-    font-size: 100%;
-  }
+  clear: both;
+}
+// Ridiculous specificity to override Blacklight
```

Comment:
I agree. Do you want to put in a pull request to fix Blacklight? We'd appreciate the help.